### PR TITLE
made todo item unique

### PR DESCRIPTION
**Pull request by akinolu52**
---
name: Unique Todo Items
about: This PR prevents double entries on todo IP items
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Solution Description

add all input items into a set and then convert it into an array

### CODE REVIEW

File: src/components/Todo/Todo.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const Todo: React.FC<TodoProps> = ({
 
     const addTodo = (e: any) => {
         e.preventDefault();
-        setLists((prevTodo) => [...prevTodo, inputValue]);
+        setLists((prevTodo) => Array.from(new Set([...prevTodo, inputValue])));
```
Comment (rayyan224) [inline src/components/Todo/Todo.tsx:43]: Goood use of sets love it

**Review by rayyan224 — APPROVED** [review]

File: src/components/Todo/Todo.tsx (hunk 1 above)
Comment (akinolu52) [reply src/components/Todo/Todo.tsx:43]: thanks

**Comment by rayyan224** [comment]
I  think we should do an error to tell the user no duplicates? Or maybe disable button ?

**Review by BillyG83 — APPROVED** [review]
smart dude! when @rayyan224 is impressed i know its good CS 😆

File: src/components/Todo/Todo.tsx (hunk 1 above)
Comment (osamajandali) [reply src/components/Todo/Todo.tsx:43]: It's a great solution indeed

**Pull request merged**